client/rest-high-level/src/main/java/org/elasticsearch/client/rollup/RollupJobCaps.java

```diff
@@ -62,7 +59,7 @@
         PARSER.declareString(ConstructingObjectParser.constructorArg(), ROLLUP_INDEX);
         PARSER.declareString(ConstructingObjectParser.constructorArg(), INDEX_PATTERN);
         PARSER.declareNamedObjects(ConstructingObjectParser.constructorArg(),
-            (p, c, name) -> new Tuple<>(name, RollupFieldCaps.fromXContent(p)), FIELDS);
+            (p, c, name) -> new AbstractMap.SimpleEntry<>(name, RollupFieldCaps.fromXContent(p)), FIELDS);
```

Comment:
Why not use `Tuple` here and then use `v1/v2` in the `Collectors.toMap` here? I'm just curious about why the switch

Reply:
ok this has been addressed in my new commit <3